docs/getting-started.md

```diff
@@ -116,7 +116,7 @@ curl.exe -L https://github.com/containerd/containerd/releases/download/v$Version
 tar.exe xvf .\containerd-windows-amd64.tar.gz
 
 # Copy and configure
-Copy-Item -Path ".\bin\*" -Destination "$Env:ProgramFiles\containerd" -Recurse -Container:$false -Force
+Copy-Item -Path ".\bin" -Destination "$Env:ProgramFiles\containerd" -Recurse -Force
```

Comment:
We don't have to remove -Container:$false. Just doing the following works:

PS C:\> Copy-Item -Path ".\bin" -Destination "$Env:ProgramFiles\containerd" -Recurse -Container:$false -Force

Reply:
We don't want to preserve the directory structure while copying to C:\ProgramFiles\containerd so I think it is important to have -Container:$false

Reply:
Ok, it's done.